internal/cli/organizations.go

```diff
@@ -461,38 +472,175 @@ auth0 orgs members ls <id>`,
-					members, apiErr := cli.api.Organization.Members(url.PathEscape(inputs.ID), opts...)
-					if apiErr != nil {
-						return nil, false, apiErr
+			members, err := cli.getOrgMembersWithSpinner(cmd.Context(), inputs.ID, inputs.Number)
+			if err != nil {
+				return err
+			}
+			sortMembers(members)
+			cli.renderer.MembersList(members)
+			return nil
+		},
+	}
+	return cmd
+}
+
+func rolesOrganizationCmd(cli *cli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "roles",
+		Short: "Manage roles of an organization",
+		Long:  "Manage roles of an organization.",
+	}
+
+	cmd.SetUsageTemplate(resourceUsageTemplate())
+	cmd.AddCommand(listRolesOrganizationCmd(cli))
+	cmd.AddCommand(membersRolesOrganizationCmd(cli))
+
+	return cmd
+}
+
+func listRolesOrganizationCmd(cli *cli) *cobra.Command {
+	var inputs struct {
+		OrgID  string
+		Number int
+	}
+
+	cmd := &cobra.Command{
+		Use:     "list",
+		Aliases: []string{"ls"},
+		Args:    cobra.MaximumNArgs(1),
+		Short:   "List roles of an organization",
+		Long:    "List roles assigned to members of an organization.",
+		Example: `auth0 orgs roles list
+auth0 orgs roles ls <id>`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) == 0 {
+				err := organizationID.Pick(cmd, &inputs.OrgID, cli.organizationPickerOptions)
+				if err != nil {
+					return err
+				}
+			} else {
+				inputs.OrgID = args[0]
+			}
+
+			members, err := cli.getOrgMembersWithSpinner(cmd.Context(), inputs.OrgID, inputs.Number)
+			if err != nil {
+				return err
+			}
+
+			roleMap := make(map[string]management.OrganizationMemberRole)
+			err = ansi.Spinner("Getting roles for each member", func() error {
+				for _, member := range members {
+					userID := member.GetUserID()
+					roleList, errInner := cli.api.Organization.MemberRoles(inputs.OrgID, userID)
+					if errInner != nil {
+						return errInner
 					}
-					var output []interface{}
-					for _, member := range members.Members {
-						output = append(output, member)
+					for _, role := range roleList.Roles {
+						roleID := role.GetID()
+						if _, exists := roleMap[roleID]; !exists {
+							roleMap[roleID] = role
+						}
 					}
-					return output, members.HasNext(), nil
-				})
-
+				}
+				return nil
+			})
 			if err != nil {
-				return fmt.Errorf("Unable to list members of an organization with Id '%s': %w", inputs.ID, err)
+				return err
 			}
-			var typedList []management.OrganizationMember
-			for _, item := range list {
-				typedList = append(typedList, item.(management.OrganizationMember))
+			// convert management.OrganizationMemberRole to management.Role
+			var roles []*management.Role
+			for _, role := range roleMap {
+				roles = append(roles, &management.Role{ID: role.ID, Name: role.Name, Description: role.Description})
 			}
-			cli.renderer.MembersList(typedList)
+			sort.Slice(roles, func(i, j int) bool {
+				return strings.ToLower(roles[i].GetName()) < strings.ToLower(roles[j].GetName())
+			})
```

Comment:
Same thing as above, would be great to move this within a func such that we don't need to add the `convert` comment but instead the func name would be self explanatory e.g. `convertOrganizationMemberRolesToRoles()`.